lib/broccoli/angular-broccoli-stylus.js

```diff
@@ -41,13 +41,13 @@ class StylusPlugin extends Plugin {
   }
 }
 
-exports.makeBroccoliTree = (sourceDir) => {
+exports.makeBroccoliTree = (sourceDir, options) => {
   if (stylus) {
     let stylusSrcTree = new Funnel(sourceDir, {
       include: ['**/*.styl'],
       allowEmpty: true
     });
 
-    return new StylusPlugin([stylusSrcTree]);
+    return new StylusPlugin([stylusSrcTree], options);
   }
 };
```

Comment:
Same as `less` and then call `stylus.render(content, this.options, function(err, css) { ..` 
Please be careful here, `this.options` should have `filename` defined.